buildspec/linuxE2ETests.yml

```diff
@@ -0,0 +1,39 @@
+version: 0.2
+
+env:
+    variables:
+        AWS_TOOLKIT_TEST_NO_COLOR: '1'
+        NO_COVERAGE: 'true'
+        # Suppress noisy apt-get/dpkg warnings like "debconf: unable to initialize frontend: Dialog").
+        DEBIAN_FRONTEND: 'noninteractive'
+        LOGIN_SECRET_ARN: 'codecatalyst-test-account'
+
+phases:
+    install:
+        commands:
+            - '>/dev/null add-apt-repository universe'
+            - '>/dev/null apt-get -qq install -y apt-transport-https'
+            - '>/dev/null apt-get -qq update'
+            - '>/dev/null apt-get -qq install -y ca-certificates'
+            - 'apt-get install --reinstall ca-certificates'
+            # Dependencies for running vscode.
+            - '>/dev/null apt-get -yqq install libatk1.0-0 libgtk-3-dev libxss1 xvfb libnss3-dev libasound2 libasound2-plugins libsecret-1-0'
```

Comment:
nit: might be time to "include" common buildspec parts to avoid duplicate code in the buildspecs

Reply:
Doesn't seem like buildspecs support 'extending' from a base file. There are other solutions of course, backlogging.